tracer/src/Datadog.AutoInstrumentation.NativeLoader/instrumented_assembly_generator/method_signature.h

```diff
@@ -0,0 +1,120 @@
+#pragma once
+#include <memory>
+#include <vector>
+#include "../com_ptr.h"
+#include "member_signature.h"
+
+namespace instrumented_assembly_generator
+{
+class MethodSignature
+{
+    PCCOR_SIGNATURE _methodSig;
```

Comment:
prefer being explicit: use `private:`

Reply:
Fixed in ebde2f57c8ef03fd167862f981abd5c4958c222a